test/bin/build_rpms.sh

```diff
@@ -119,13 +120,22 @@ download_brew_rpms() {
     mkdir -p "${BREW_RPM_SOURCE}"
     if "${SCRIPTDIR}/manage_brew_rpms.sh" access ; then
```

Comment:
@ggiguash  This block is not accessible from CI on cache-nightly runs, [see this log](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/logs/periodic-ci-openshift-microshift-release-4.19-periodics-e2e-aws-tests-cache-nightly/1933320950179172352/artifacts/e2e-aws-tests-cache-nightly/openshift-microshift-infra-iso-build/build-log.txt):
```
+ 00:55:44.906969816 ./bin/build_rpms.sh:130 	echo 'WARNING: The Brew Hub site is not accessible, skipping the download'
```

For that reason, I also implemented this logic in the CI script, see https://github.com/openshift/release/pull/66024 PR

Reply:
You are right. Sorry for misleading you.

Reply:
I'm sorry, I don't understand what you mean by `necessary version downloads`

Reply:
I meant adding the following commands [here](https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/microshift/infra/iso-build/openshift-microshift-infra-iso-build-commands.sh#L56) - pseudo code:
```
        bash -x ./test/bin/manage_brew_rpms.sh download "${ocpversion}" "${out_path}"
        bash -x ./test/bin/manage_brew_rpms.sh download "${ocpversion}" - 1 "${out_path}"
        bash -x ./test/bin/manage_brew_rpms.sh download "${ocpversion}" - 2 "${out_path}"
```

Reply:
done here: https://github.com/openshift/microshift/pull/4900/files#diff-c5558625fd237b08c74b6c9a3a55f7d994792bebf3c4a9ee2548a96d18810db4R130-R136